packages/edit-site/src/components/start-template-options/style.scss

```diff
@@ -1,3 +1,31 @@
+.edit-site-start-template-options__modal {
+	.components-modal__content {
+		padding-bottom: 0;
+	}
+
+	.components-modal__children-container {
+		display: flex;
+		height: 100%;
+		flex-direction: column;
```

Comment:
Posting this 
The pattern of using internal components class names should be avoided at all costs. Classnames should be treated as private, inaccessible APIs outside of a given component. We should try to work towards reducing this technical debt, instead of adding more instances of it.

Here are a few alternatives that come to mind, in this scenario:
- If this need is specific to this use case, we could add a wrapper `div` around the `children` of Modal, give it a custom classname, and assign those same styles (similarly to what happens for `.edit-site-start-template-options__modal__actions`)
- if styling the content wrapper is a frequent need, we could consider adding something like a `contentWrapperClassname` prop to `Modal`
- in the long run, we should consider exporting `Modal` in a more modular fashion, allowing consumers of the component to assemble the component and tweak it more granularly (this is not the top priority for us right now, but if you'd like to work on it, we're happy to help)

cc @mirka

Reply:
>If this need is specific to this use case, we could add a wrapper div around the children of Modal, give it a custom classname,

We can't do that because even we set our wrapper to be `height:100%` the parent div(where I added the `components-modal__children-container` css class) won't be, making it ineffective.

>we could consider adding something like a contentWrapperClassname prop to Modal
 
This is good alternative for now, yes.

Reply:
> > we could consider adding something like a contentWrapperClassname prop to Modal
> 
> This is good alternative for now, yes.

Quick check — @mirka , do you see any issues with going for this approach? Or do you have any other approach in mind?

Reply:
I might be misunderstanding the requirement, but couldn't this be achieved via the existing className prop? It's just about the height, no?

```jsx
<Modal className="my-modal" />
```
```css
.my-modal {
  height: 100%;
}
```

Reply:
Hey @ntsekouras , would it be possible to open a follow-up PR?

- we should remove the reference to the `components-modal__content` classname
- we should remove the addition of the `components-modal__children-container` classname to `Modal`, and rework the styles. Would lena's suggestion work?

Thank you! 🙏

Reply:
>Would lena's suggestion work?

No, because similarly: 

>We can't do that because even we set our wrapper to be height:100% the parent div(where I added the components-modal__children-container css class) won't be, making it ineffective.

>we should remove the addition of the components-modal__children-container classname to Modal, and rework the styles

You mean change the design not to be in the bottom or just target that div with something like the `>` selector?

Reply:
Opened #50655 for a possible approach that doesn't involve internal `Modal` class names